api/v1alpha1/environment.go

```diff
@@ -136,7 +136,7 @@ func GetEnvironmentsByNames(environments []*Environment, names []string) []*Envi
 	return envs
 }
 
-func (e *Environment) IsPromotableTo(targetEnv *Environment) bool {
+func (e Environment) IsPromotableTo(targetEnv *Environment) bool {
```

Comment:
If we are unifying the receiver names, can we agree to prefer short but non letter names? I think `env.Spec` reads much better than `e.Spec`.

The only exception I believe in is the actual conventions like `w` for io.Writer, `r` for io.Reader, b for `bytes.Buffer` etc.

Reply:
I personally go for letters with receivers but I'm fine with both

Reply:
I'm OK with that, but we already have a huge amount of 1-letter receivers! 🤷